collections/ansible_collections/cloudkit/config_as_code/README.md

```diff
@@ -201,21 +209,24 @@ oc apply -f cloudkit_env.yml -n fulfillment-aap
 
 #### Option 1: from your local environment
 
-Set your environment (same as above in `<your AAP organization>-<your AAP
-project>-config-as-code-ig` secret):
+Create an Ansible var file with the following content:
+```
+---
+aap_hostname: ...
+aap_username: ...
+aap_password: ...
+aap_organization_name: ...
+aap_project_name: ...
+aap_project_git_uri: ...
+aap_project_git_branch: ...
+aap_ee_image: ...
+aap_validate_certs: ...
+license_manifest_path: ...
+```
```

Comment:
This seems to substantially duplicate the information on line 183. Do we need to have this in two separate places?